syntax = "proto3";
package glproto;
service View {
  rpc ClearColor(ClearColorRequest) returns (ClearColorReply) {}
}
rpc ClearColor(ClearColorRequest) returns (ClearColorReply) {};
message ClearColorRequest {
  float red = 0;
  float green = 1;
  float blue = 2;
  float alpha = 3;
}
message ClearColorReply {
  bool success = 0;
  float elapsed_ms = 1;
}